lib/cadence/crew.rb

```diff
@@ -31,12 +31,23 @@ def stop(signal)
       crew.each { |pid| stop_worker(signal, pid) }
     end
 
+    def after_fork(block)
```

Comment:
nit: you might wanna allow `&block` so you can also do this:

```ruby
worker.after_fork do |worker|
  worker.register_workflow(MyWorkflow)
end
```

Reply:
Thanks!